lib/tasks/scrapeMovies.rake

```diff
@@ -0,0 +1,75 @@
+namespace :scrapeMovies do 
+  require "net/http"
+  require "uri"
+
+
+  desc "scrape oscars.org for list of best movies"
+  task :movies => :environment do |t, args|
+
+    require 'open-uri'
+    require 'nokogiri'
+
+    #  can call like this if you want
+    # rake scrapeMovies:movies year="2000"
+    year = ENV['year']
+    # puts "year is #{year}"
+
+    for year in 1940..2015 do
+
+      url = "http://www.imdb.com/event/ev0000003/" + year.to_s
+
+
+      begin
+        document = open(url).read
+        html_doc = Nokogiri::HTML(document)
+        puts html_doc.css('title').text
+      rescue OpenURI::HTTPError => ex
+        puts "Missing URL"
+        return
+      end
+
+
+      data_movies = "blockquote > blockquote:nth-child(2) > div > strong:nth-child(2) > a:nth-child(1)"
+      movies = html_doc.css(data_movies)
+
+      count = movies.count
+      if count == 0 then
+        puts "big problem - no movies"
+        return
```

Comment:
I would prefer this scrape stops if there is a missing page so I think return is ok..?